assets/sounds/se/se.json

```diff
@@ -140,5 +140,27 @@
                 }
             }
         }
+    },
+    "pound1": {
```

Comment:
Please join these sfx with the `psynergy` section. You can use Audacity to join them.